internal/provider/data_source_project.go

```diff
@@ -28,6 +28,11 @@ func dataSourceTFEProject() *schema.Resource {
 				Required: true,
 			},
 
+			"description": {
```

Comment:
@netramali if you're adding an attribute to a data source we should note this in the changelog - see [example](https://github.com/hashicorp/terraform-provider-tfe/blob/main/CHANGELOG.md#v0500)

Reply:
I got a suggestion to remove it. I have added it back again.